### PR TITLE
Make Jacoco settings flexible

### PR DESCRIPTION
**Pull request by hasys**

### CODE REVIEW

**Comment by hasys** [comment]
Hi @psiroky,
these changes will be in use for fast surface test-coverage analysis in child repositories (initially I plan to use it in kie-wb-common-stunner).
Thank you

File: pom.xml [hunk 1]
```diff
@@ -598,15 +601,15 @@
               </goals>
               <configuration>
                 <!-- Do not fail the build if rules are not met -->
```
Comment (psiroky) [inline pom.xml]: This comment should be removed as the value is not hardcoded anymore.

**Comment by hasys** [comment]
@psiroky , thank you!

rebased. + version also moved to configurations. So now we can play with junit+mock+jacoco combination

File: pom.xml [hunk 2]
```diff
@@ -145,6 +145,10 @@
     <version.bundle.plugin>2.5.3</version.bundle.plugin>
     <!-- Surefire version 2.19.1 coming from parent is buggy and kie-osgi tests in droolsjbpm-integration are failing because of that. -->
     <version.surefire.plugin>2.18.1</version.surefire.plugin>
+    <!-- Jacoco plugin configurations -->
+    <jacoco.haltOnFailure>false</jacoco.haltOnFailure>
+    <jacoco.limit.minimum>0.9</jacoco.limit.minimum>
```
Comment (psiroky) [inline pom.xml]: Sorry for not noticing before, but this should have a more descriptive name. What about `jacoco.lineCoverageLimit`?

**Comment by hasys** [comment]
@psiroky, done.

**Comment by psiroky** [comment]
Merged by https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/commit/87461ee4547461c590a8ae7c6594dae96da8004d

**Pull request closed without merging**